src/main/resources/application.properties

```diff
@@ -7,4 +7,7 @@ spring.datasource.url=${datasource.formplayer.url}
 spring.datasource.username=${datasource.formplayer.username}
 spring.datasource.password=${datasource.formplayer.password}
 
-commcare.formplayer.purge.trigger=never
\ No newline at end of file
+commcare.formplayer.purge.trigger=never
+commcare.formplayer.scheduledTasks.purge.cron=0 0 0 * * *
```

Comment:
I'm not familiar with the `resources/application.properties` but do we have to insert this in ansible as well?

Reply:
Hi Kevin,

The settings in `resources/application.properties` are the default config for the app, so as long as they are conservative and function without and external dependencies they don't need to be set in ansible.

It may be the case that we would want to run a different set of parameters for production than we would for a 'standard' install of formplayer (significantly more or significantly less conservative), in which case we'd want to add them to ansible.